ansible_runner/runner.py

```diff
@@ -22,8 +22,8 @@
 
 class Runner(object):
 
-    def __init__(self, config, cancel_callback=None, remove_partials=True,
-                 event_handler=None, finished_callback=None, status_handler=None):
+    def __init__(self, config, cancel_callback=None, event_handler=None,
```

Comment:
Partial deletion control is also a parameterization of the module that we'll need to remove also.

Reply:
Where?

Reply:
hmm, I guess I'm wrong about this one.